git_runner.py

```diff
@@ -7,7 +7,7 @@ def __init__(self, timeline):
         self.timeline = timeline
 
     def run(self, command_list):
-        if command_list[0] == 'commit':
+        if command_list[0] == 'commit' or command_list[0] == 'add':
```

Comment:
Eventually, I think the logic of recording command or not should be in `add.py`/`command.py` etc...